src/Akka.Streams.Kafka/reference.conf

```diff
@@ -35,7 +35,7 @@ akka.kafka.consumer {
   
   # Tuning property that indicates how many records to poll from the Kafka consumer
   # in any given attempt.
-  max-poll-records = 10
+  max-poll-records = 500
```

Comment:
Set this to 500, which has consistently yielded the best results from our benchmarks (because it's equal to the internal poll max-buffer-size)